packages/x-components/src/components/base-dropdown.vue

```diff
@@ -34,6 +34,7 @@
         :aria-expanded="isOpen.toString()"
         class="x-dropdown__items-list"
         role="listbox"
+        tabIndex="-1"
```

Comment:
Should be 0? 

[TabIndex](https://developer.mozilla.org/es/docs/Web/HTML/Global_attributes/tabindex)